README.md

```diff
@@ -85,6 +85,23 @@ jobs:
 ```
 
 ## Publishing using Apache Maven
+
+### Extra setup for pom.xml:
+
+According to possible issues with `Inappropriate ioctl for device` or `gpg: signing failed: No such file or directory`, Maven GPG Plugin configuration in pom.xml should contain the following:
```

Comment:
Maven GPG Plugin configuration in `pom.xml` file should contain the following structure to avoid possible issues like `Inappropriate ioctl for device` or `gpg: signing failed: No such file or directory`:

Reply:
Done. Thank you.